airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -144,6 +144,10 @@ def _latest_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.buck
     latest_path = get_metadata_remote_file_path(metadata.data.dockerRepository, "latest")
     return upload_file_if_changed(metadata_file_path, bucket, latest_path, disable_cache=True)
 
+def _release_candidate_upload(metadata: ConnectorMetadataDefinitionV0, bucket: storage.bucket.Bucket, metadata_file_path: Path) -> Tuple[bool, str]:
```

Comment:
This looks almost identical to `_latest_upload` on line 143. Should we move "release_candidate" (and "latest", from line 144) to constants, and then have just a helper function `_upload` that takes it in as a variable, instead of two helper functions?